integration_test/Utils.ts

```diff
@@ -0,0 +1,411 @@
+import Web3 from 'web3';
+import * as web3Utils from 'web3-utils';
+import * as fs from 'fs';
+import * as path from 'path';
+import BigNumber from 'bignumber.js';
+import { interacts } from '@openst/mosaic-contracts';
+import { EIP20Token } from '@openst/mosaic-contracts/dist/interacts/EIP20Token';
+import { TransactionObject } from '@openst/mosaic-contracts/dist/interacts/types';
+import { Organization } from '@openst/mosaic-contracts/dist/interacts/Organization';
+import { OSTComposer } from '@openst/mosaic-contracts/dist/interacts/OSTComposer';
+import Repositories from '../src/repositories/Repositories';
+import Directory from '../src/Directory';
+import StakeRequest from '../src/models/StakeRequest';
+import { MessageStatus } from '../src/repositories/MessageRepository';
+import assert from '../test/test_utils/assert';
+import MosaicConfig from '../src/Config/MosaicConfig';
+import { default as SrcUtils } from '../src/Utils';
+import GraphClient from '../src/subscriptions/GraphClient';
+
+const EthUtils = require('ethereumjs-util');
+
+const mosaicConfig = MosaicConfig.fromFile(path.join(__dirname, 'mosaic.json'));
+
+const ABIDirectoryPath = path.join(__dirname, 'abi');
+
+const auxSubGraphRpc = 'http://127.0.0.1:11000/subgraphs/name/mosaic/auxiliary-1000';
+
+const fetchQuery = 'query ($contractAddress: Bytes!, $messageHash: Bytes!) {\n'
+  + 'mintProgresseds(orderBy: uts, orderDirection: asc, first: 1, where:'
+  + ' {contractAddress: $contractAddress, _messageHash: $messageHash}) {\n'
+  + '    id\n'
+  + '    _messageHash\n'
+  + '    _staker\n'
+  + '    _beneficiary\n'
+  + '    _stakeAmount\n'
+  + '    _mintedAmount\n'
+  + '    _rewardAmount\n'
+  + '    _proofProgress\n'
+  + '    _unlockSecret\n'
+  + '    contractAddress\n'
+  + '    blockNumber\n'
+  + '    uts\n'
+  + '  }\n'
+  + '}';
+
+/**
+ * It contains common helper methods to test facilitator.
+ */
+export default class Utils {
+  public originWeb3: Web3;
+
+  public auxiliaryWeb3: Web3;
+
+  private originFunder: string;
+
+  private auxiliaryFunder: string;
+
+  private ostComposer: string;
+
+  private step: number;
```

Comment:
should step be enum to make it more readable.

Reply:
Done.